packages/service/src/domain/commands/processors/playback-current-command.ts

```diff
@@ -36,10 +36,12 @@ class PlaybackCurrentCommand extends CommandProcessor {
     const queueSongsTitles = includeQueue ? queue.map((song) => `  - ${song.name.truncated(50, false)}`) : [];
     const queueTitle = !queueSongsTitles.isEmpty() ? 'W kolejce:' : 'Playlista jest pusta';
     const currentSongName = current?.name ?? '';
+    const currentYoutubeId = current?.youtubeId ?? '';
 
     const text = [
       currentSongName ? 'Obecnie gramy:' : 'Obecnie nic nie gramy.',
       currentSongName ? `  ${currentSongName}` : '',
+      currentYoutubeId ? `  https://www.youtube.com/watch?v=${currentYoutubeId}` : '',
```

Comment:
Skoro w 39 linii robisz `?? ''` to tutaj ten ternary operator chyba nie potrzebny?